insights/specs/default.py

```diff
@@ -351,6 +351,7 @@ def dumpdev(broker):
     grubby_default_kernel = simple_command("/usr/sbin/grubby --default-kernel")  # RHEL6 and updwards
     hammer_ping = simple_command("/usr/bin/hammer ping")
     hammer_task_list = simple_command("/usr/bin/hammer --config /root/.hammer/cli.modules.d/foreman.yml --output csv task list --search 'state=running AND ( label=Actions::Candlepin::ListenOnCandlepinEvents OR label=Actions::Katello::EventQueue::Monitor )'")
+    hammer_compute_resource_list = simple_command("hammer --interactive 0 --output json compute-resource list")
```

Comment:
Since the command will be executed by python but without a shell, please specify it's its full path to avoid cannot find the `hammer` command`. 

`/usr/bin/hammer`

Reply:
fixed